tenant/src/main/scala/com/improving/app/tenant/TenantServiceImpl.scala

```diff
@@ -24,7 +24,8 @@ class TenantServiceImpl(sys: ActorSystem[_]) extends TenantService {
   sharding.init(Entity(Tenant.TypeKey)(
     createBehavior = entityContext =>
       Tenant(
-        PersistenceId(entityContext.entityTypeKey.name, entityContext.entityId)
+        PersistenceId(entityContext.entityTypeKey.name, entityContext.entityId),
+        new Info() // TODO: obtain correct Info
```

Comment:
I'm not clear on how the initialization of the entity actor should be handled here if we want these actors to always have complete data.